releasenotes/2.1.2.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.2
+date: 2018-02-06 20:00:00
+release_notes: true
+permalink: /releasenotes/2.1.2/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.2 is the second bugfix release after jclouds 2.1.0 and comes with several bugfixes and performance improvements. 
+You can [read the official announcement here](https://s.apache.org/jclouds212). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+New features in Apache jclouds 2.1.2 include:
+
+* 1&1 Baremetal servers
+* EC2 r5, t3, and x1 instance types
+* GCS REGIONAL storage class
+
+### Bugs and patches
+
+* [JCLOUDS-1330](https://issues.apache.org/jira/browse/JCLOUDS-1330) - Azure ARM orphaned networks not cleaned up when node deleted
```

Comment:
Is this list accurate? E.g. [JCLOUDS-1330](https://issues.apache.org/jira/browse/JCLOUDS-1330) is still marked as "open", and the list of issues [from JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314430&version=12343896) looks quite different?

Reply:
> I accidentally included unresolved issues which marked 2.1.2 as the fix version

Ah, clear, thanks!